### PR TITLE
fix(bucket): make bucket id and relation type required for relation property

### PR DESCRIPTION
**Pull request by htuna07**

### CODE REVIEW

File: stacks/spica/src/bucket/components/relation-schema/relation-schema.component.ts [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
+      if (!this.schema.relationType) {
+        this.schema.relationType = RelationType.OneToOne;
+      }
+      this.buckets = this.bucketService.getBuckets().pipe(
+        tap(buckets => {
+          if (!this.schema.bucketId) {
+            this.schema.bucketId = buckets[0]._id;
+          }
+        })
+      );
```
Comment (thesayyn) [inline stacks/spica/src/bucket/components/relation-schema/relation-schema.component.ts:33]: You can put these into constructor or ngOnInit. then you wouldn't need to use setTimeout.

Comment (htuna07) [reply stacks/spica/src/bucket/components/relation-schema/relation-schema.component.ts:33]: **Expression has changed after it was checked** error shows up when I made these changes.

Comment (thesayyn) [reply stacks/spica/src/bucket/components/relation-schema/relation-schema.component.ts:33]: How about doing it in the constructor?

Comment (htuna07) [reply stacks/spica/src/bucket/components/relation-schema/relation-schema.component.ts:33]: Same error.

Comment (htuna07) [reply stacks/spica/src/bucket/components/relation-schema/relation-schema.component.ts:33]: It works in the constructor with setTimeout.

**Review by thesayyn — APPROVED** [review]

**Pull request merged**